src/main/java/seedu/address/logic/commands/ListCommand.java

```diff
@@ -18,9 +23,32 @@
     public static final String COMMAND_WORD = "list";
 
     public static final String MESSAGE_SUCCESS_DECK = "Listed all decks";
-    public static final String MESSAGE_SUCCESS_CARD = "Listed all cards";
+    public static final String MESSAGE_SUCCESS_ALL_CARDS = "Listed all cards";
+    public static final String MESSAGE_SUCCESS_PERFORMANCE_CARDS = "Listed %1$s cards";
     public static final String AUTOCOMPLETE_TEXT = COMMAND_WORD;
```

Comment:
You might want to update the autocomplete text to have a default difficulty

Reply:
Unless, of course, the preference is for the autocomplete to default to listing all the cards. In which case just leave it as it is

Reply:
I think it would be more intuitive to have the default case be listing all cards!